### DISCUSSION TITLE
After upgrade from 1.8.1 to 1.8.2 get only empty page

### DISCUSSION THREAD

**Discussion by Andyt8** [YOURLS/YOURLS#3062] [Q&A] [answered]
I have a really strange behaviour. I was previously using YOURLS 1.8.1 with PHP8 on Apache2.

If I upgrade to 1.8.2, I only get empty pages. No title, no content. I can find the status code 500 by pressing F12 and reaccess the website.

If I search in the log files on the web server, there is no (new) entry. I mean no PHP error or anything else is recorded. 
If I use a deliberately faulty php file or deactivate a necessary PHP module for another website, then I see these errors normally in the log file.

Since according to the blog article there shouldn't be that much difference between 1.8.1 and 1.8.2, this confuses me even more.

Please for help.

**Comment by ozh** [comment]
Most likely you didn't overwrite properly one or several files.

**Comment by Andyt8** [comment]
That is a good point. I have done the whole process twice. Checked it several times afterwards. Either I always miss the same thing or it has another reason.

I had also already removed all folders except the "user" folder and re-uploaded from 1.8.2. The result was the same. Maybe something from the "user" folder?

**Reply by ozh** [reply]
Good practice : if something doesn't work as expected, disabling plugins (renaming the plugin directory is a good way) cannot hurt if the culprit is there.

General rule is : try to pinpoint everything that is specific to your setup (YOURLS : config, plugins ; Apache : .htaccess ; etc...)

**Comment by Andyt8** [comment]
Thank you for your fast response. I found nothing about plugins and htaccess - tried rename and remove. The old config looks good and a comparision with the newest config template doesn't show me anything new. I know, the access is working, because returning to 1.8.1 work without troubles.

I also tried running only 1.8.2 files with my config and it is still the same. Show me only empty pages and no messages in any log files. What a strange behaviour. It seems to me, it is the same if there is also no config file - I mean it look like it stop before trying to read the config file...

At the moment I don't know what to try next. Maybe I will try to run 1.8.2 with a new database. It is like fishing in the mud without any information from any log file.

**Comment by Andyt8** [comment, marked as answer]
I have fixed the problem. By the way, if the config.php file was missing, I got an error message in the log. If it was back, there was no error message. However, I found the option to **enable YOURLS debug mode** and with that, I got an error message... - yeh

The problem was somehow strange and a little mistake by myself. But after I could read the error message, I was able to solve the matter quickly.

It work now with the latest version.
